package.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0"?>
 <package>
   <name>rosjava_bootstrap</name>
-  <version>0.3.2</version>
+  <version>0.3.3</version>
```

Comment:
Would you please remove this change? We usually bump the version in a separate commit with a short description in the changelog.

If this code is already working for you, I can do that right after this PR.